manuscript/03.results.md

```diff
@@ -4,16 +4,15 @@
 
 ![**Population-based measurements are insufficient to distinguish the structure of heterogeneous populations.**  Selected lineages of AU565 cells (a) untreated or (b) treated with 5 nM gemcitabine. Each line indicates the lifetime of one cell. The line branching into two lines indicates cell division. The G1 and S/G2 phase durations are shown in solid thick and thin lines, respectively.](figure1.svg){#fig:lineage}
 
 ### A lineage tree-based hidden Markov model infers the state of cells given measurements on lineage trees
 
 <!-- ASM proofed. -->
 
-Given the unique insights that single cell measurements on lineage trees can provide, we wished to implement a strategy for classifying cells based on their phenotype and relationships. Given a set of measurements made across a lineage tree, we used a lineage tree-based hidden Markov model (tHMM) to fit these data (Fig. {@fig:tree}a). Like a typical hidden Markov model, a tHMM can infer the hidden discrete states of cells given a series of measurements where a state could be defined by specific phenotype distributions. This inference takes place using an iterative strategy wherein the states of each cell are predicted given the phenotype of not only the cell but also all other cells in their lineage (“expectation” step), and then each distribution of phenotypes is fit to match the cells within that state (“maximization” step) (Fig. {@fig:tree}b). This expectation-maximization (EM) process repeats until convergence.
-
+Given the unique insights that single cell measurements on lineage trees can provide, we wished to implement a strategy for classifying cells based on their phenotype and relationships. Given a set of measurements made across a lineage tree, we used a lineage tree-based hidden Markov model (tHMM) to fit these data (Fig. {@fig:tree}a). Like a typical hidden Markov model, a tHMM can infer the hidden discrete "states" of cells given a series of measurements where a state could be defined by specific phenotype distributions. In this study, cells that belong to a "state" are a subpopulation of cells that share a common trait; for instance, cells that are highly arrested in their G1 phase with high concentrations of a drug, could belong to one state. Of note, these states are only identified based on measurable observations from an experiment. The inference of these states takes place using an iterative strategy wherein the states of each cell are predicted given the phenotype of not only the cell but also all other cells in their lineage (“expectation” step), and then each distribution of phenotypes is fit to match the cells within that state (“maximization” step) (Fig. {@fig:tree}b). This expectation-maximization (EM) process repeats until convergenc.
```

Comment:
Do you mean for the two paragraphs to be merged here? There must be a blank line between paragraphs for them to remain separate.

Reply:
I didn't mean to do that. I just fixed it. Also fixed the misspelling of the last word.